.gitignore

```diff
@@ -9,6 +9,7 @@
 *.o
 *.obj
 *.od
+*.ssexwsp
```

Comment:
Ow... that's a project file for a text editor / mini-IDE I'm writing in my spare time.
Screenshot here: http://labs.pragmaware.net/misc/ssex.shot.1.jpg
And... guess what... it uses ctags for symbol tagging :)
I'll probably release it to the public one day, once it's finished.

I can remove that line from .gitignore, but it shouldn't hurt anyone if it stays there..